invenio_oauthclient/templates/semantic-ui/invenio_oauthclient/login_user.html

```diff
@@ -0,0 +1,24 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2015-2018 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+{%- extends config.OAUTHCLIENT_LOGIN_USER_TEMPLATE_PARENT %}
+
+{%- from "invenio_oauthclient/_macros.html" import oauth_button %}
+
+
+{%- block form_outer %}
+  {% if config.OAUTHCLIENT_REMOTE_APPS %}
+  <div class="ui form">
+    {% for name in config.OAUTHCLIENT_REMOTE_APPS.keys() %}
+      {{ oauth_button(name, next=request.args.get('next')) }}
+    {% endfor %}
+  </div>
+    <h3 align="center">&mdash; OR &mdash;</h3>
```

Comment:
maybe you can use the [Semantic UI divider here](https://semantic-ui.com/elements/divider.html)?